src/ServiceControl.Monitoring/Http/HttpEndpoint.cs

```diff
@@ -52,7 +53,7 @@ class NancyTask : FeatureStartupTask
 
             public NancyTask(IBuilder builder, Uri host)
             {
-                var buildstrapper = new Bootstrapper(builder.BuildAll<IEndpointDataProvider>());
+                var buildstrapper = new Bootstrapper(builder.BuildAll<IEndpointDataProvider>(), builder.Build<DiagramDataProvider>());
```

Comment:
Maybe we should invest in making it impl-agnostic and based only on the all data providers? With this approach, there will be an additional registration per provider.